src/components/UI/NetworkSwap/NetworkSwap.js

```diff
@@ -14,7 +14,7 @@ export const NetworkSwap = ({isFlipped, onClick}) => {
     <div className={toClasses(styles.networkSwap, isFlipped && styles.flipped)} onClick={onClick}>
       <Circle
         color={colorDarkSlateBlue}
-        size={50}
+        size={32}
```

Comment:
use CircleLogoSize, and this is too small IMO.

Reply:
done per Figma